sotodlib/obs_ops/splits.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+from sotodlib.core import FlagManager
+
+def det_splits_relative(aman, det_left_right=False, det_upper_lower=False, det_in_out=False):
```

Comment:
Add an argument called `wrap` which defaults to `None` and if None or False it does not wrap the flags into the input axis manager. If `wrap` is a True it wraps into some default field name `det_flags` and if `wrap` is a string then it wraps the  flags into that field name.